examples/fargate-serverless/main.tf

```diff
@@ -103,6 +94,9 @@ module "eks_blueprints_kubernetes_addons" {
   eks_oidc_provider    = module.eks_blueprints.oidc_provider
   eks_cluster_version  = module.eks_blueprints.eks_cluster_version
 
+  # Wait on the `kube-system` profile before provisioning addons
+  data_plane_wait_arn = module.eks_blueprints.fargate_profiles["kube_system"].eks_fargate_profile_arn
```

Comment:
pretty nifty - addons don't start provisioning until the profile has provisioned

Reply:
do you find this approach better than the use of `depends_on` ?
I assume it replaces it, corrrect?